task/types.go

```diff
@@ -65,6 +63,9 @@ type Repository struct {
 	Download string `json:"download"`
 }
 
+// ExecutableUrls is a map of os/arch to URLs where the binary executable can be downloaded
+type ExecutableUrls map[string]string
```

Comment:
i'd prefer to define more clearly in a struct

type ExecutableDownload {
  typeOfExecutable string
  urls map[string]string
}